### DISCUSSION TITLE
Ability to use URL in URL bar for Grafana link

### DISCUSSION THREAD

**Discussion by junderw** [teslamate-org/teslamate#1892] [Ideas]
Use case:

There might be a case where you have multiple ways of accessing teslamate. ie. Tor Hidden Service and public facing NGINX reverse proxy etc.

Problem:

Currently, the setting page only allows one value for a URL to grafana.

Idea:

choose some non-URI-compliant string as a stand-in for "the protocol + domain currently in the URL bar"

ie. The setting `%%WEBROOT%%:3000` would make grafana link as follows:
1. Access from tor via `http://xxxxxxxxx.onion` or `http://xxxxxxxxx.onion:4000` will make the grafana link become `http://xxxxxxxxx.onion:3000`
2. Access from NGINX via `https://mydoimain.com` or `https://mydoimain.com:4000` will make the grafana link become `https://mydoimain.com:3000`

(I don't know if %% would be appropriate, I'm just showing the concept)

**Comment by LukeSkywalker993** [comment]
hi, do u have a solution for this?
